src/helpers/helpers.lut.js

```diff
@@ -45,16 +45,16 @@ export default class HelpersLut {
 
   initCanvasContainer(dom) {
     let canvasContainer = dom;
-    canvasContainer.style.width = '256 px';
-    canvasContainer.style.height = '128 px';
     canvasContainer.style.border = '1px solid #F9F9F9';
     return canvasContainer;
   }
 
   createCanvas() {
     let canvas = document.createElement('canvas');
-    canvas.height = 16;
+    canvas.height = 1;
     canvas.width = 256;
+    canvas.style.width = '256px';
+    canvas.style.height = '16px';
```

Comment:
shouldn't `canvas.height = 1;` and `canvas.style.height = '16px';` or do they have different meanings (i.e. `canvas.height` is the size of the actual canvas and `canvas.style.height` the representation of the same canvas in the DOM?)

Reply:
Yes, height is the actual size of the pixel when painting to it, whereas the style of the Dom element is how it gets rendered.